yt/visualization/plot_modifications.py

```diff
@@ -105,7 +105,7 @@ def project_coords(self, plot, coord):
         code_length units.  Projected data units are 2D versions of the
         simulation data units relative to the axes of the final plot.
         """
-        if len(coord) == 3:
+        if coord.shape[0] == 3:
             if not isinstance(coord, YTArray):
```

Comment:
We could maybe put this outside the conditional, which would fix my question above...

Reply:
The concern here was that some arrays *shouldn't* be `YTArray` objects for dealing with plot, figure, or axis coord systems.  But this should now be addressed in the code.  The switch to `coord.shape[0]` from `len(coord)` was residual from a previous iteration at trying to fix this issue.  Either should be fine here because I'm sanitizing the inputs, but I swapped it back to `len(coord)` just to minimize code changes and make things simpler.